app/scripts/vendor/env-test.js

```diff
@@ -31,13 +31,8 @@
   var pwElement = document.createElement('input');
   pwElement.setAttribute('type', 'password');
 
-  try {
-    // IE8 will blow up here.
-    pwElement.setAttribute('type', 'text');
-    docElement.className += ' toggle-pw';
-  } catch(e) {
-    docElement.className += ' no-toggle-pw';
-  }
+  pwElement.setAttribute('type', 'text');
+  docElement.className += ' toggle-pw';
```

Comment:
if we are removing `no-toggle-pw`, then we can also remove the `no-toggle-pw` css rule.
@shane-tomlinson could you confirm that we do not need it?